### PR TITLE
handle utc date conversion correctly

### PR DESCRIPTION
**Pull request by ptouchton**
## Original issue(s)
department-of-veterans-affairs/va.gov-team#36488

## Testing done
Manual and unit

## Screenshots
image

## Acceptance criteria
- [ ] Requested dates should be displayed on the day that was chosen when making a CC request.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by vbahinwillit — CHANGES_REQUESTED** [review]
The following unit test is failing consistently for me:

      AssertionError: expected body to contain 'Thu, February 24, 2022 in the afternoon', but the text was 'HomeHealth careSchedule and manage health appointmentsVA online schedulingRequest detailPending primary care appointmentThe time and date of this appointment are still to be determined.VA appointmentCheyenne VA Medical Center2360 East Pershing BoulevardCheyenne, WyomingWY 82001-5356DirectionsMain phone: 307-778-7550TTY: 711Preferred type of appointmentOffice visitPreferred date and timeFri, February 25, 2022 in the afternoonSat, February 26, 2022 in the afternoonYou shared these details about your concernNew issue: A message from the patientYour contact detailsEmail: veteranemailtest@va.govPhone number: 212-555-12122 1 2. 5 5 5. 1 2 1 2.Call morningCancel Request'
      + expected - actual

      -HomeHealth careSchedule and manage health appointmentsVA online schedulingRequest detailPending primary care appointmentThe time and date of this appointment are still to be determined.VA appointmentCheyenne VA Medical Center2360 East Pershing BoulevardCheyenne, WyomingWY 82001-5356DirectionsMain phone: 307-778-7550TTY: 711Preferred type of appointmentOffice visitPreferred date and timeFri, February 25, 2022 in the afternoonSat, February 26, 2022 in the afternoonYou shared these details about your concernNew issue: A message from the patientYour contact detailsEmail: veteranemailtest@va.govPhone number: 212-555-12122 1 2. 5 5 5. 1 2 1 2.Call morningCancel Request
      +Thu, February 24, 2022 in the afternoon
      
      at _callee10$ (src/applications/vaos/tests/appointment-list/components/RequestedAppointmentDetailsPage.unit.spec.jsx:726:43)
      at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)
      at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)
      at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)
      at asyncGeneratorStep (src/applications/vaos/tests/appointment-list/components/RequestedAppointmentDetailsPage.unit.spec.jsx:39:103)
      at _next (src/applications/vaos/tests/appointment-list/components/RequestedAppointmentDetailsPage.unit.spec.jsx:41:194)

**Comment by ptouchton** [comment]
@vbahinwillit - That test that is failing for you is one I didn't touch. Would you mind running it in the master branch and see if you get the same result?

**Review by jenniemc — APPROVED** [review]
LGTM!

**Comment by vbahinwillit** [comment]
Got the following error when running the unit test:

```
 -HomeHealth careSchedule and manage health appointmentsVA online schedulingRequest detailPending primary care appointmentThe time and date of this appointment are still to be determined.VA appointmentCheyenne VA Medical Center2360 East Pershing BoulevardCheyenne, WyomingWY 82001-5356DirectionsMain phone: 307-778-7550TTY: 711Preferred type of appointmentOffice visitPreferred date and timeFri, March 4, 2022 in the afternoonSat, March 5, 2022 in the afternoonYou shared these details about your concernNew issue: A message from the patientYour contact detailsEmail: veteranemailtest@va.govPhone number: 212-555-12122 1 2. 5 5 5. 1 2 1 2.Call morningCancel Request
      +Thu, March 3, 2022 in the afternoon
      
      at _callee10$ (src/applications/vaos/tests/appointment-list/components/RequestedAppointmentDetailsPage.unit.spec.jsx:726:43)
      at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)
      at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)
      at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)
      at asyncGeneratorStep (src/applications/vaos/tests/appointment-list/components/RequestedAppointmentDetailsPage.unit.spec.jsx:39:103)
      at _next (src/applications/vaos/tests/appointment-list/components/RequestedAppointmentDetailsPage.unit.spec.jsx:41:194)
```

**Review by vbahinwillit — APPROVED** [review]

File: src/applications/vaos/tests/appointment-list/components/RequestedAppointmentDetailsPage.unit.spec.jsx [hunk 1]
```diff
@@ -724,14 +724,14 @@ describe('VAOS <RequestedAppointmentDetailsPage> with VAOS service', () => {
...
+        appointment.attributes.requestedPeriods[0].start.replace('Z', ''),
+      ).format('ddd, MMMM D, YYYY')} in the afternoon`,
     );
     expect(screen.baseElement).to.contain.text(
-      `${moment(appointment.attributes.requestedPeriods[1].start).format(
-        'ddd, MMMM D, YYYY',
-      )} in the afternoon`,
+      `${moment(
+        appointment.attributes.requestedPeriods[1].start.replace('Z', ''),
+      ).format('ddd, MMMM D, YYYY')} in the afternoon`,
```
[the remark this answers is not in the document]
Comment (ptouchton) [reply src/applications/vaos/tests/appointment-list/components/RequestedAppointmentDetailsPage.unit.spec.jsx]: Tony, I'm thinking there are still some issues with the version of code you have locally. In the latest commit these are the lines for that span


image

**Pull request merged**